src/Microsoft.DotNet.Darc/src/Darc/Options/VirtualMonoRepo/VmrCommandLineOptions.cs

```diff
@@ -26,26 +26,25 @@ public IServiceCollection RegisterServices()
         var tmpPath = Path.GetFullPath(TmpPath ?? Path.GetTempPath());
         string gitHubToken = null;
         string azureDevOpsToken = null;
+        LocalSettings localDarcSettings = null;
 
         try
         {
-            var localDarcSettings = LocalSettings.LoadSettingsFile(this);
-            gitHubToken = GitHubPat ?? localDarcSettings?.GitHubToken;
-            azureDevOpsToken = AzureDevOpsPat ?? localDarcSettings?.AzureDevOpsToken;
+            localDarcSettings = LocalSettings.LoadSettingsFile(this);
         }
         catch (DarcException)
         {
-
+            // we want to allow null values for GitHubToken and AzureDevOpsToken 
```

Comment:
@MilenaHristova my 2 cents - when adding comments, try to put in them what is not in the code around. The code quite clearly reads as "we don't need to have the tokens" so it's better to explain why (which can have some non-code related justification).

```suggestion
            // The VMR synchronization often works with public repositories where tokens are not required
```